### ISSUE TITLE
New-UDColumn - UI inconstancy 

### ISSUE THREAD

**Issue by Phyziik** [ironmansoftware/universal-dashboard#101]
I was trying to understand how the new UDColumn parameters worked, and I was pretty confused as to how they work.
Here is what I tried : 

```
$homePage = New-UDPage -Name "Home" -Content {
   New-UDRow -Columns {
      New-UDColumn -SmallSize 2 -DebugEndpoint -Endpoint {
         New-UDCard -Title "Test Card" -Content {"$testString"} -Reveal {New-UDCard -Title "I appeared"}
      }
   }
   New-UDRow -Columns {
      New-UDColumn -MediumSize 2 -DebugEndpoint -Endpoint {
         New-UDCard -Title "Test Card" -Content {"$testString"} -Reveal {New-UDCard -Title "I appeared"}
      }
   }
   New-UDRow -Columns {
      New-UDColumn -LargeSize 2 -DebugEndpoint -Endpoint {
         New-UDCard -Title "Test Card" -Content {"$testString"} -Reveal {New-UDCard -Title "I appeared"}
      }
   }
}
$redirector = New-UDPage -Url "/ComputerMonitoring/:testString" -Endpoint {
   param($testString)
   New-UDCard -Title "Test Card" -Content {"$testString"} -Reveal ""test
}
$MyDashboard = New-UDDashboard -Title "Hello, World" -Pages @($homePage,$redirector) -EndpointInitializationScript {
   Enable-UDLogging -Level Error -FilePath ".\debug_logger.txt" -Console
}
Start-UDDashboard -Port 1000 -Dashboard $MyDashboard -AutoReload
```

And here is what I got : 
![image](https://user-images.githubusercontent.com/37376232/39202671-97db2e44-47c0-11e8-9894-e7b7234dd95f.png)

Might be a misunderstanding on my side, but I feel like the parameters are broken.

**Comment by adamdriscoll** [comment]
You are correct. The large size is broken. I'm using the $MediumSize variable for the LargeSize.

https://github.com/ironmansoftware/ud-controls/blob/master/src/column.ps1#L51

The idea with the different sizes is based on screen size. 

The small size will be used on screens smaller than 600px wide. 
The medium size will be used on screens smaller than 992px wide and larger than 600px wide. 
The large size will be used on screens larger than 992px wide. 

I'll get that all into the documentation before the 1.6.0 release.

**Comment by adamdriscoll** [comment]
All these sizes come from the UI library I'm using for UD: http://materializecss.com/grid.html

**Comment by adamdriscoll** [comment]
This has been fixed in 1.6.0-beta3.
